lib/skate/application.ex

```diff
@@ -16,6 +16,7 @@ defmodule Skate.Application do
       SkateWeb.Endpoint,
       # Starts a worker by calling: Skate.Worker.start_link(arg)
       # {Skate.Worker, arg},
+      worker(HealthWorker, []),
```

Comment:
Now that HealthWorker and GTFS are linked, you should also change the strategy on line 25 to `:one_for_all`. Otherwise, if the GTFS worker crashes, the HealthWorker won't know about it and will continue to report that the system is healthy.